lib/devise_token_auth/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 module DeviseTokenAuth
-  VERSION = '1.1.3'.freeze
+  VERSION = '1.1.4'.freeze
```

Comment:
why this change?

Reply:
I guess it doesn't make sense. Not used to making contributions to public repos.

Reply:
```suggestion
  VERSION = '1.1.3'.freeze
```